src/Components/Web.JS/src/Platform/BootConfig.ts

```diff
@@ -17,8 +24,17 @@ export class BootConfigResult {
 
     return new BootConfigResult(bootConfig, applicationEnvironment);
   };
+
+  public static async defaultLoadBlazorBootJson(url: string) : Promise<Response> {
```

Comment:
Looks like this could just be a regular `function` - doesn't need to be publicly reachable from `BootConfigResult` AFAICT.